src/cc_catalog_airflow/dags/provider_api_scripts/brooklyn_museum.py

```diff
@@ -0,0 +1,214 @@
+"""
+Content Provider:       Brooklyn Museum
+
+ETL Process:            Use the API to identify all CC licensed artworks.
+
+Output:                 TSV file containing the images and respective meta-data.
+
+Notes:                  https://www.brooklynmuseum.org/opencollection/api
+                        3000 calls per day (per API Key)
+"""
+import logging
+import os
+import re
+from common.requester import DelayedRequester
+from common.storage import image
+
+logger = logging.getLogger(__name__)
+
+DELAY = 3.0  # time delay (in seconds)
+RETRIES = 3
+API_KEY = os.getenv('BROOKLYN_MUSEUM_API_KEY')
+PROVIDER = 'brooklynmuseum'
+LIMIT = 10
+ENDPOINT = 'https://www.brooklynmuseum.org/api/v2/'
+
+logging.basicConfig(format='%(asctime)s: [%(levelname)s - Brooklyn Museum API] =======> %(message)s', level=logging.INFO)
```

Comment:
Please use the `logging.basicConfig` style from the other new Provider API Scripts, e.g., `flickr.py`.